sdk/eventhub/ci.legacy.yml

```diff
@@ -0,0 +1,55 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
```

Comment:
We have tried to avoid using terms like legacy. If we are to follow our other naming conventions we should call these ci.data.yml to represent the track 1 data plane libraries.

Reply:
In this case, "legacy" is the terminology that was agreed upon with the service team.   I'd prefer to keep the nomenclature that is used within the ecosystem to avoid confusion.  When I looked through the pipelines, most services with multiples appear to be split between "data" and "mgmt";  I never would have guessed that "data" was referring to T1.

Reply:
yeah we were trying to keep "empty" (i.e. ci.yml) as the forward looking track 2 set, also sometimes called client and then for any track1 data plane we were using "ci.data.yml" or "ci.mgmt.yml" for mgmt.

Reply:
If I'm being honest, I find that pattern confusing, but I don't feel strongly enough to swim upstream.  I'll adjust to the recommendation.

Reply:
There is no great convention but I believe it is better then using the term "legacy" and it at least has some consistency across different areas and languages.